src/actinia_core/processing/actinia_processing/ephemeral/persistent_processing.py

```diff
@@ -25,21 +25,20 @@
 Asynchronous computation in specific temporary generated and then copied
 or original mapsets
```

Comment:
Same here.

Reply:
I think this discribes only the persistent processng in first a temporary mapset and the the mapset merge